arches/app/templates/views/graph/graph-settings.htm

```diff
@@ -154,7 +154,7 @@ <h3 class="panel-title card-main-title">{% trans "Settings" %}</h3>
                                                         </div>
 
                                                         <div class="col-xs-12">
-                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: graph.ontology_id, valueAllowUnset: true, options: ontologies, optionsText: 'name', optionsValue:'ontologyid', optionsCaption: '{% trans "No ontology" %}', chosen: {disable_search_threshold: 10, width: '100%'}"></select>
+                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: graph.ontology_id, valueAllowUnset: true, options: ontologies, optionsText: {% trans 'name' %}, optionsValue:'ontologyid', optionsCaption: {% trans 'No ontology' %}, chosen: {disable_search_threshold: 10, width: '100%'}"></select>
```

Comment:
**optionsText** should NOT be wrapped in a trans tag, as this is a keyword in code